udata/core/dataset/rdf.py

```diff
@@ -400,6 +403,8 @@ def dataset_from_rdf(graph, dataset=None, node=None):
     dataset.title = rdf_value(d, DCT.title)
     dataset.description = sanitize_html(d.value(DCT.description))
     dataset.frequency = frequency_from_rdf(d.value(DCT.accrualPeriodicity))
+    dataset.created_at = rdf_value(d, DCT.issued, dataset.created_at)
+    dataset.last_modified = rdf_value(d, DCT.modified, dataset.last_modified)
```

Comment:
A few thoughts/questions about those:
- no safeguard? Ie, is it a date, is it defined...
- more meta: do we still have a "machine" created and modified dates available on datasets if we override those? I don't think we can afford to loose those totally.

Reply:
If undefined, the default value is used in  `rdf_value`. But indeed, there is no additional safeguard. A similar logic is used [for resources](https://github.com/opendatateam/udata/blob/7f8f9b8b6d6882f2fd51d48021a5b672bf9e7ab1/udata/core/dataset/rdf.py#L380).

I'm not clear with the different dates properties but it seemed to be the case for resources in DCAT and also in [udata-ods](https://github.com/opendatateam/udata-ods/blob/168ec81c097ffa06a362aaff977b79317329996b/udata_ods/harvesters.py#L158) and [udata-ckan](https://github.com/opendatateam/udata-ckan/blob/a7c5f4e67311152b066d697fc8899d5941b1f6d4/udata_ckan/harvesters.py#L155) as well. I went with the same logic, but indeed, we can discuss this in https://github.com/etalab/data.gouv.fr/issues/720